cloudify_cli/commands/audit_log.py

```diff
@@ -31,18 +37,46 @@ def auditlog():
 @cfy.pass_client()
 def list_logs(creator_name,
               execution_id,
+              since,
+              follow,
+              timeout,
               sort_by,
               descending,
               pagination_offset,
               pagination_size,
               logger,
               client,
               ):
+    since_timestamp = before_to_utc_timestamp(since) if since else None
+    if follow:
+        raise CloudifyCliError('Streaming requires Python>=3.6.')
```

Comment:
Assuming this command gets overridden on 3.6+ by the asyncio one, perhaps this should just not accept the follow arg, and instead have a comment in its help indicating that python should be upgraded to get more functionality?

Reply:
this is more visible though. I can imagine users calling support to ask "why don't I have streaming support???", because they didn't read the helptxt. But this is impossible to miss.

Reply:
Fair enough- it just looked a bit strange at first glance, but makes sense.